src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -175,6 +176,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        ILTransform
+
+    Extension point for IL transformations.
+    ============================================================
+    -->
+  <Target Name="ILTransform"
+          Condition="'$(_TargetFrameworkVersionWithoutV)' >= '3.0' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'"
+          DependsOnTargets="$(ILTransformDependsOn)" />
```

Comment:
Why is this only for `.NetCoreApp`? How about, if there are whole unused IL files that are not reachable from any public entrypoints in a `.netstandard` dll?  (ex: partial classes) ?

Reply:
I think you're right that it would make sense for netstandard, and the linker behavior is probably similar to the portable app case.

Reply:
I think that since linking happens during publish, the normal workflow of packaging a netstandard library would not run the linker, and it would take some work by library authors to package a linked library. I agree that it makes sense not to block it here.